src/Utils/JsonHelper.jsx

```diff
@@ -5,7 +5,26 @@ DOMPurify.setConfig({ADD_ATTR: ['target']});
 const lang = document.documentElement.lang;
 const locale = lang !== "en" ? `/${lang}` : '';
 
+//fetch fields.json, pages.json, cards.json
 export const fetchData = async (filename, setContent, setError) => {
+  const path = `https://vote.gov${locale}/nvrf/assets/${filename}`;
+  const response = await fetch(path)
+    .then(response => response.json())
+    .catch(() => setError(true));
+  setContent(response);
+}
+
+//fetch [state]/data.json
+export const fetchStateData = async (state, setContent, setError) => {
+  const path = `https://vote.gov${locale}/nvrf/assets/state/${state}/data.json`;
+  const response = await fetch(path)
+    .then(response => response.json())
+    .catch(() => setError(true));
+  setContent(response);
+}
+
+//fetch local strings.json file - need to test fetching from vote.gov
+export const fetchStringData = async (filename, setContent, setError) => {
```

Comment:
@rayestrada I added this in to fetch strings.json locally. I am having issues with CORS for this file url only. This shouldn't be the case, because Tyler is allowing all JSON for all the allowed origins. Any idea why this file would be having issues?

Reply:
I am removing the local strings file so we can pull from vote.gov.